lib/RAST_SDK/RAST_SDKImpl.pm

```diff
@@ -1469,6 +1470,13 @@ sub annotate_genomes
 	    call_features_prophage_phispy => 1,
 	    retain_old_anno_for_hypotheticals => 1
 	});
+	
+	my $empty_input_msg = ("Required inputs: genomes in an array with at least one genome;\n".
+	                       "and/or a list of genome names separated by ';', '\n' or '|' (without quotes).\n";
```

Comment:
Stray `(`:
```
my $empty_input_msg = ("
```

Reply:
Users won't know what is meant by 'genomes in an array with at least one genome'. because they don't know how the data is represented internal to the app.

How about the phrase: "must select at least one genome"?

Reply:
Error message improved.